core/region.go

```diff
@@ -377,14 +377,14 @@ func (r *region) findMatchingASGsInPageOfResults(groups []*autoscaling.Group,
 		// expression. The goal is to add the matching ASGs when running in opt-in
 		// mode and the other way round.
 		if optInFilterMode != groupMatchesExpectedTags {
-			logger.Printf("Skipping group %s because its tags, the currently "+
+			debug.Printf("Skipping group %s because its tags, the currently "+
 				"configured filtering mode (%s) and tag filters do not align\n",
 				asgName, r.conf.TagFilteringMode)
 			continue
 		}
 
 		if stackName := getTagValueFromASGWithMatchingTag(group, tagCloudFormationStackName); stackName != nil {
-			logger.Println("Stack: ", *stackName)
+			debug.Println("Stack: ", *stackName)
```

Comment:
We log if the stack is updating. If it's not updating, the stack isn't that interesting.